modules/manage/pages/cluster-maintenance/cluster-balancing.adoc

```diff
@@ -9,6 +9,30 @@ With an Enterprise license, you can additionally enable _Continuous Data Balanci
 
 Cluster balancing protects you from unbalanced systems that saturate resources on one or more brokers. This can affect throughput and latency. Furthermore, a cluster with replicas on a down broker risks availability loss if more brokers fail, and a cluster that keeps losing brokers without healing eventually risks data loss.
 
+The following table summarizes the various balancing options in Redpanda:
+
+
+.Cluster balancing options
+[cols="3,1,6"]
+|===
+|Balancer |License |Description
+
+|Leadership balancing
+|Community
+Enterprise
+|Topic-aware balancer designed to distribute partition leadership across available nodes. This helps avoid topic leadership hotspots on one or a few specific nodes in your cluster.
+
+|Partition replica balancing
```

Comment:
Partition Replica Balancing is probably a clearer name than what we have below (Redpanda Partition Balancing ) ... might be worth changing the section name to be Partition Replica Balancing?